### ISSUE TITLE
Berserker mayhem

### ISSUE THREAD

**Issue by V-i-x-x** [any-other-guy/Endless-Chaos-LostArk#7]
Hey Man

I love the repo and the python code that you did +rep😊 and appreciate sharing it and updating it.

I was wondering did you ever get banned using this on a test account?

And did you have a change to test it on berserker?

I would test it out but I am banned for a month lol. My main is a berserker with mayhem build.

Thank you 😊

Regards
Victor

**Comment by any-other-guy** [comment]
Hello, thank you for such kind words.

I started coding the bot on the first day since I reached T3 in May 2022, been using the bot almost 24x7 on my main steam and mutiple alt steam accounts ever since,. More than 2000+ hours of play time accumulated.

And yes I got banned and unbanned already for buying gold on G2G. The email AGS sent me specifically mentioned the amount of gold that I RMTed, therefore I assumed that I was solely banned for buyinig gold.

As for berserker, I personally have not played the class. The logic of how the bot works is pretty generic for different classes I would say. With a appropriate setting, I believe the logic could handle most classes, some performs better some worse. However, you will have to carefully configure a couple of things in [config.py](../blob/main/config.py) and [abilities.py](../blob/main/abilities.py). I recommend to use abilities that does large aoe around the character itself, and also use Heavy Armor and Protection rune if possible at the beginning.

Cheers.
